src/Tokenizer/Tokens.php

```diff
@@ -319,10 +319,7 @@ public function offsetSet($index, $newval): void
             $this->changed = true;
             $this->namespaceDeclarations = null;
 
-            if (isset($this[$index])) {
```

Comment:
why removing? what if $this[$index] is not existing?
i wouldn't like to pass null to unregisterFoundToken

Reply:
Personally I think we should unregister using `$index`, not `$this[$index]`. Then it would be always `int` and condition is not needed. `unregisterFoundToken()` internally should check if there's something under specified index and do the required logic, otherwise do nothing.

I accepted current changes because I did not see that `null` is not accepted in `unregisterFoundToken()`, but the idea of common place for such check remains the same and I was OK with it. Using index instead the problem is solved 😉.